firmware/config/stm32f7ems/efifeatures.h

```diff
@@ -6,262 +6,75 @@
  * @date Aug 29, 2013
  * @author Andrey Belomutskiy, (c) 2012-2017
  */
+ 
+#include "../stm32f4ems/efifeatures.h"
 
-#ifndef EFIFEATURES_H_
-#define EFIFEATURES_H_
+#ifndef EFIFEATURES_STM32F7_H_
+#define EFIFEATURES_STM32F7_H_
 
-#define EFI_GPIO_HARDWARE TRUE
+// Warning! This is a test config!
 
-#define EFI_FSIO TRUE
-
-#define EFI_PWM_TESTER FALSE
-
-#define HAL_USE_USB_MSD FALSE
-
```

Comment:
weird how PR page shows a major removal here, but I guess that's just because you've copied & edited file?

Reply:
I've added some comments inside efifeatures.h on this matter.